index.d.ts

```diff
@@ -303,6 +303,21 @@ declare module '@primer/components' {
     Link: React.FunctionComponent<UnderlineNavLinkProps>
```

Comment:
We'll need to add a type declaration for `FilteredSearch` as well

Reply:
Yep, still a todo